File: DataSend1903F1/app/src/main/java/com/example/datasend1903f1/MainActivity.java
package com.example.datasend1903f1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
EditText etname;
Button login;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        etname=findViewById(R.id.editTextTextPersonName);
        login =findViewById(R.id.button);
    }
    public void CheckUser(View v){
        String name=etname.getText().toString();
        if(name.equals("")){
            Toast.makeText(getApplicationContext(),"Enter Name First",Toast.LENGTH_LONG).show();
        }
        else{
            Intent i=new Intent(MainActivity.this,DashBoradActivity.class);
            i.putExtra("sendName",name);
            startActivity(i);
        }


    }
}

File: FirstKotlinProject/app/src/main/java/com/example/firstkotlinproject/MainActivity.kt
package com.example.firstkotlinproject

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var btn=findViewById<Button>(R.id.button)
        btn.setOnClickListener(){
            Toast.makeText(this,"This is First Message",Toast.LENGTH_LONG).show();
        }

        var btnNext=findViewById<Button>(R.id.button1)
        btnNext.setOnClickListener(){
            var i =Intent(this,MainActivity2::class.java)
            startActivity(i)
        }
    }
}